src/cdk/table/table.md

```diff
@@ -135,6 +135,14 @@ updates. This can be based on _anything_: websocket connections, user interactio
 time-based intervals, etc. Most commonly, updates will be triggered by user interactions like
 sorting and pagination.
 
+##### `fixedLayout`
+Enabling this option will enforce uniform column widths and optimize the rendering of sticky
```

Comment:
We should mention that this option only applies to native `<table>` elements and not to the flexbox-based `<cdk-table>`. 

Would it also be possible to expand on this explanation a bit with details on why this is valuable for optimizing sticky elements?

Reply:
Done